src/plugins/ssh/index.ts

```diff
@@ -51,6 +51,8 @@ const SUDO_MESSAGE = /Sorry, user .+ may not run sudo on .+/; // The output of `
  */
 const DEFAULT_VALIDATION_WINDOW_MS = 5e3;
 
+const RETRY_DELAY_MS = 3000;
```

Comment:
```suggestion
const RETRY_DELAY_MS = 500;
```

Idea being that we avoid continuous retries (If one attempt is really quick this is enough) but we are still fast in detecting access once it's provisioned